src/chrome/content/rules/amd-osx.com.xml

```diff
@@ -0,0 +1,14 @@
+<ruleset name="AMD OS X">
+	<target host="amd-osx.com" />
+	<target host="*.amd-osx.com" />
+
+	<securecookie host="^([\w.-]+)\.amd-osx\.com$" name=".+" />
+
+	<test url="http://forum.amd-osx.com/" />
+	<test url="http://amd-osx.com/" />	
+	<test url="http://forum.amd-osx.com/search.php?search_id=active_topics" />
+	<test url="http://forum.amd-osx.com/memberlist.php?mode=contactadmin/" />
+
+	<rule from="^http://([\w.-]+)\.amd-osx\.com/"
+		to="https://$1.amd-osx.com/" />
```

Comment:
We prefer `<rule from="^http:" to="https:" />` for simple rules.

Reply:
Thanks, fixed.